### DISCUSSION TITLE
General Question

### DISCUSSION THREAD

**Discussion by Gta211** [community/community#145947] [Mobile]
How can I hide the "Shortcut's" section on the Home Tab on the mobile app on iOS? <details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.187.0 (147865513)
Device: iPhone 14 Pro Max (iOS 18.1.1)
</details>

**Comment by tyagraj777** [comment]
Thanks for this question


Unfortunately, the "Shortcuts" section on the Home Tab of the GitHub mobile app for iOS cannot be hidden or customized as of the latest version (1.187.0). GitHub does not currently offer options to rearrange or remove sections from the Home Tab within the app.

"Workarounds"
Here are some potential actions you can take:

1] Provide Feedback to GitHub

- If hiding or customizing the "Shortcuts" section is important to you, consider submitting feedback directly to GitHub:

1. Tap your profile icon in the app.
2. Navigate to Settings > Send Feedback.
3. Suggest adding the ability to hide or customize the Home Tab.

2] Use GitHub on a Browser

- If the "Shortcuts" section is a distraction, consider using GitHub in a mobile browser where the interface is different and doesn't include the same sections as the mobile app.

3]Focus on Other Tabs

- You can navigate to other tabs like Notifications, Pull Requests, or Issues to avoid seeing the Home Tab.

GitHub frequently updates its mobile app, so a future release may include the ability to customize the Home Tab. Keep an eye on the GitHub Mobile Release Notes for updates!

All the Best !!
